motr/io_pargrp.c

```diff
@@ -1988,13 +1989,15 @@ static int pargrp_iomap_dgmode_recover(struct pargrp_iomap *map)
 		rc = -EIO;
 		goto end;
 	}
+#if !ISAL_ENCODE_ENABLED
```

Comment:
I have comments on similar issues in kernel implementation.

Reply:
Done.